deps/build.jl

```diff
@@ -15,6 +15,15 @@ director_version = v"0.1.0"
         director = library_dependency("ddApp", aliases=["libddApp"], depends=[python_vtk, python])
     ]
 
+    linux_distributor = strip(readstring(`lsb_release -i -s`))
+    linux_version = VersionNumber(strip(readstring(`lsb_release -r -s`)))
+    if linux_distributor == "Ubuntu" && linux_version >= v"16.04"
+        director_variant = "ubuntu16"
+    else
+        director_variant = "linux"
```

Comment:
@rdeits i'll probably add ubuntu14 to the other package name, too.  I'll update you when that happens.  Thanks for being an early consumer of the binaries!